### PR TITLE
Add a safeguard when a region collapsible has a null or not

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
On the live site, it shows an "Onbekend" in the collapsible. Didn't want to remove the functionality completely so I've added a check when one or more region includes a `null` in the data, it should show depending on that. (and then we can add the appreciate text in Lokalise for example if needed).

Problem:
Screenshot 2021-06-22 at 16 48 15

### CODE REVIEW

File: packages/app/src/domain/topical/escalation-level-explanations.tsx [hunk 1]
```diff
@@ -29,8 +29,13 @@ function EscalationLevelExplanation(props: EscalationLevelExplanationProps) {
     </Box>
   );
 }
+interface escalationLevelExplanationsProps {
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/topical/escalation-level-explanations.tsx:32]: Try to pronounce this 10x really quickly 😅

File: packages/app/src/domain/topical/escalation-level-explanations.tsx [hunk 2]
```diff
@@ -29,8 +29,13 @@ function EscalationLevelExplanation(props: EscalationLevelExplanationProps) {
...
+  hasUnknown: boolean;
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/domain/topical/escalation-level-explanations.tsx:33]: I'm thinking "has unknown what?" 🤔  level?

Comment (VWSCoronaDashboard8) [reply packages/app/src/domain/topical/escalation-level-explanations.tsx:33]: I'd suggest showUnknownLevel or hasUnknownLevel.

File: packages/app/src/pages/actueel/gemeente/[code].tsx [hunk 3]
```diff
@@ -359,7 +359,13 @@ const TopicalMunicipality = (props: StaticProps<typeof getStaticProps>) => {
               </ChoroplethTwoColumnLayout>
 
               <Box mt={4}>
-                <EscalationLevelExplanations />
+                <EscalationLevelExplanations
+                  hasUnknown={
+                    choropleth.vr.escalation_levels.filter(
+                      (item) => item.level === null
+                    ).length > 0
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/actueel/gemeente/[code].tsx:366]:
```suggestion
                    some(choropleth.vr.escalation_levels, x => !isPresent(x))
```

I think this should work using lodash some and ts-is-present

File: packages/app/src/pages/actueel/veiligheidsregio/[code].tsx [hunk 4]
```diff
@@ -323,7 +323,13 @@ const TopicalSafetyRegion = (props: StaticProps<typeof getStaticProps>) => {
               </ChoroplethTwoColumnLayout>
 
               <Box mt={4}>
-                <EscalationLevelExplanations />
+                <EscalationLevelExplanations
+                  hasUnknown={
+                    choropleth.vr.escalation_levels.filter(
+                      (item) => item.level === null
+                    ).length > 0
+                  }
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/pages/actueel/veiligheidsregio/[code].tsx:331]: Same here of course

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Pull request merged**